packages/client/hmi-client/src/page/ModelEditor.vue

```diff
@@ -60,13 +60,12 @@
 </template>
 
 <script setup lang="ts">
-import * as d3 from 'd3';
 import _ from 'lodash';
-import graphScaffolder, { IGraph } from '@graph-scaffolder/index';
-import { runDagreLayout, D3SelectionINode, D3SelectionIEdge } from '@/services/graph';
+import { IGraph } from '@graph-scaffolder/index';
```

Comment:
Can we do `{ IGraph as GraphType }` to be consistent with our coding style?

Reply:
I would avoid this kind of in-situ renaming of things of the sake of consistency: graph-scaffolder is it's own stand-alone library and should be treated as such in my opinion, like, we would not do `type HTMLElementType = HTMLElement`.

If you think that graph-scaffolder itself should conform to terarium naming standard, that will need to be a different PR.